ui-v2/src/api/flow-runs/index.ts

```diff
@@ -217,3 +217,93 @@ export const useDeploymentCreateFlowRun = () => {
 		...rest,
 	};
 };
+
+/**
+ * Request body for setting a flow run state
+ */
+type SetFlowRunStateParams = {
+	/**
+	 * The ID of the flow run to update
+	 */
+	id: string;
+	/**
+	 * The new state type to set
+	 */
+	state: components["schemas"]["StateType"];
+	/**
+	 * Optional name for the state
+	 */
+	name?: string | null;
+	/**
+	 * Optional message to associate with the state change
+	 */
+	message?: string | null;
+	/**
+	 * Whether to force the state change, bypassing orchestration rules
+	 */
+	force?: boolean;
+};
```

Comment:
You should be able to get this type from `components["schemas"]` which will simplify the signature of `useSetFlowRunState`. It might have a kinda crazy name though, so an alias would be good.